code/modules/clothing/head/hardhat.dm

```diff
@@ -81,4 +81,90 @@
 	heat_protection = HEAD
 	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
 	cold_protection = HEAD
-	min_cold_protection_temperature = FIRE_HELM_MIN_TEMP_PROTECT
\ No newline at end of file
+	min_cold_protection_temperature = FIRE_HELM_MIN_TEMP_PROTECT
+
+/obj/item/clothing/head/hardhat/weldhat/

```

Comment:
/obj/item/clothing/head/hardhat/weldhat/ -> /obj/item/clothing/head/hardhat/weldhat

Reply:
Just remove the spare / at the end and by the ***LOOKS*** it should pass ya goof

Reply:
Thanks

Reply:
Looking at it, runtiming with white one?

Reply:
Thanks for pointing it out, I think I got it.